src/OpenTelemetry.Instrumentation.Process/ProcessMetrics.cs

```diff
@@ -49,6 +51,18 @@ static ProcessMetrics()
             },
             unit: "By",
             description: "The amount of committed virtual memory.");
+
+        // TODO: change to ObservableUpDownCounter
+        MeterInstance.CreateObservableGauge(
```

Comment:
I didn't see this instrument being mentioned in the OTel Process spec.

https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/process-metrics.md#process

If you think it is a a important instrument to have for ProcessMetrics, you could consider to create a pull request on the spec.